modules/oe_translation_block_field/src/BlockFieldProcessor.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Drupal\oe_translation_block_field;
+
+use Drupal\block_field\BlockFieldItemInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Render\Element;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\tmgmt_content\DefaultFieldProcessor;
+
+/**
+ * Field processor for the Block reference field.
+ */
+class BlockFieldProcessor extends DefaultFieldProcessor {
+
+  use StringTranslationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function extractTranslatableData(FieldItemListInterface $field) {
+    $data = [];
+    /* @var \Drupal\Core\Field\FieldItemInterface $field_item */
```

Comment:
Probably should be `\Drupal\Core\Field\FieldDefinitionInterface $field_definition`